testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -713,11 +714,11 @@ def setUp(self):
         # http://www.wwpdb.org/documentation/format32/sect9.html#ATOM
         self.prec = 3
 
-    def test_uses_Biopython(self):
+    def test_uses_PDBReader(self):
```

Comment:
Check that we don't have the same test setup with the old primitivePDBReader/Writer. If that is the case you can delete this test completely.